config/v1/types_feature.go

```diff
@@ -71,6 +71,8 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 			"ExperimentalCriticalPodAnnotation", // sig-pod, sjenning
 			"RotateKubeletServerCertificate",    // sig-pod, sjenning
 			"SupportPodPidsLimit",               // sig-pod, sjenning
+			"CSIBlockVolume",                    // sig-storage, j-griffith
+			"BlockVolume",                       // sig-storage, j-griffith
```

Comment:
certainly a NAK on these changes. We are not going to make these available by default out of the box.

Reply:
That's certainly fair, I'll remove them from the default set; there was some confusion (on my part) whether they were required to be there or not, I will remove if we can reach agreement on the remaining piece.  Thanks!